Packs/AWS-Enrichment-Remediation/Scripts/AWSRecreateSG/AWSRecreateSG.py

```diff
@@ -86,13 +86,6 @@ def sg_fix(sg_info: list, port: int, protocol: str, assume_role: str, instance_t
                     and rule['IpProtocol'] == protocol
                 ):
                     change = True
-                elif (
```

Comment:
this is not a duplicate, please don't remove

Reply:
This elif is when the offending rule references the port exactly.  The next elif is when the port i part of a range the therefore we need to run it past the split_rule() function.

Reply:
The `elif` block I removed is a duplicate of the `if` block before it not the `elif` block after it.

Reply:
you are right, good catch